action.yml

```diff
@@ -23,6 +23,12 @@ inputs:
   runtime:
     description: "The runtime to use for evals. Valid values: node, python."
     required: true
+  package_manager:
+    description:
+      "The package manager to use for evals. Valid values: npm, pnpm, yarn, pip,
+      or uv depending on the runtime."
+    required: false
+    default: ""
```

Comment:
Should we define a default? it looks like if package_manager is "", it goes into an empty case statement

Reply:
yeah it's a bit odd, but it's for backward compatibility. I could try without the default '', but ultimately in the code it will fall back to '' due to the zod parsing

Reply:
ahh gotchu! makes sense